### ISSUE TITLE
HTTPS Redirect

### ISSUE THREAD

**Issue by jordanliu** [google/wikiloop-battlefield#208]
http://battlefield.wikiloop.org/ does not redirect to https

![image](https://user-images.githubusercontent.com/9423525/78970592-46087600-7acf-11ea-8593-dd968f6d245e.png)

**Comment by xinbenlv** [comment]
Yes, we are working on rolling out the HTTPS support. Some additional works needs to be done.

Thank you for filing the bug.
